webapps/camunda-bpm-sdk-js/lib/api-client/resources/telemetry.js

```diff
@@ -62,4 +62,17 @@ Telemetry.configure = function(payload, done) {
   });
 };
 
+/**
+ * Fetches telemetry data for diagnostics.
+ *
+ * @param  {Object}   payload
+ * @param  {Function} done
+ */
+ Telemetry.fetchData = function(payload, done) {
+  return this.http.get(this.path + '/data', {
```

Comment:
Can we make this API call behave more gracefully in terms of the absence of the `payload` parameter?

Proposal:
```suggestion
  // allows to pass only a callback
  if (typeof params === 'function') {
    done = params;
    params = {};
  }
  params = params || {};
  done = done || noop;
  
  return this.http.get(this.path + '/data', {
```